nbf/Makefile

```diff
@@ -0,0 +1,47 @@
+TOP ?= $(shell git rev-parse --show-toplevel)
+
+export BP_SDK_DIR ?= $(TOP)/sdk
+export BP_RTL_DIR ?= $(TOP)/rtl
+
+include $(BP_SDK_DIR)/Makefile.common
+
+BP_COMMON_DIR ?= $(BP_RTL_DIR)/bp_common
+
+RISCV_OBJDUMP ?= $(CROSS_COMPILE)objdump -d -t
+RISCV_OBJCOPY ?= $(CROSS_COMPILE)objcopy
+MEM2NBF       ?= $(BP_COMMON_DIR)/software/py/nbf.py
+
+SED           ?= sed
+
+# only unicore configurations are currently supported
+NCPUS       := 1
+PADDR_WIDTH := 33
```

Comment:
I think this should be 40-bits to match the standard BP unicore config.

Reply:
Oh, yeah, looks like this is wrong. Will-fix.